config/runtime.exs

```diff
@@ -3,7 +3,8 @@ import Config
 if config_env() == :prod do
   config :slax, Slax.Repo,
     url: System.get_env("DATABASE_URL"),
-    pool_size: String.to_integer(System.get_env("POOL_SIZE") || "10")
+    pool_size: String.to_integer(System.get_env("POOL_SIZE") || "10"),
+    queue_target: 5000
```

Comment:
Can we follow the pattern above so it can be tweaked through environment variables? We should also update the `POOL_SIZE` env var in Heroku to be a bit lower to reduce the # of connections in the pool, maybe back down to 10